pennylane/data/data_manager.py

```diff
@@ -197,7 +197,11 @@ def _generate_folders(node, folders):
     """
 
     next_folders = folders[1:]
-    folders = set(node) if folders[0] == ["full"] else set(folders[0]).intersection(set(node))
+    if folders[0] == ["full"]:
+        folders = node
```

Comment:
Do we have to maintain any order when users mention "full"?

Reply:
we should maintain _some_ order, whatever it is. This always uses the order found in the data_struct.json file in S3.

Reply:
I think the order found in `data_struct` should be sufficient.

Reply:
By the way I was looking for this option "full" and couldnt find it in the docs (and tried "all" for lols) The way I did it in my example was run it for one example that doesnt exist and then take the list from the error message 🥷

Reply:
I'll add a note to the docstring for `qml.data.load` in another PR. FWIW, it is documented but only in the [high-level datasets intro doc](https://docs.pennylane.ai/en/stable/introduction/data.html#loading-datasets-in-pennylane)

Reply:
Ahhh I remember seeing that page but didnt find it, I was looking on the datasets website and the dataset module. When I am _using_ the datasets I usually just spend time on the [qchem site](https://pennylane.ai/qml/datasets_qchem.html)

Reply:
Just wanted to suggest linking that page on the qchem site, but it already is 🤐 🥵